articles/container-registry/container-registry-import-images.md

```diff
@@ -361,6 +361,15 @@ az acr import \
 Import-AzContainerRegistryImage -RegistryName myregistry -ResourceGroupName myResourceGroup -SourceRegistryUri docker.io/sourcerepo -SourceImage sourcerrepo:tag -Username <username> -Password <password>
 ```
 
+### Troubleshoot Import Container Images
+#### Symptoms and Causes
+- `The remote server may not be RFC 7233 compliant`
+  - [Distribution-spec](https://github.com/opencontainers/distribution-spec/blob/main/spec.md) is based on the [specification for the Docker Registry HTTP API V2 protocol]. [Docker Registry HTTP API V2 spec](https://github.com/docker/distribution/blob/5cb406d511b7b9163bff9b6439072e4892e5ae3b/docs/spec/api.md) allows range header form of `Range: bytes=<start>-<end>`. However, the remote server may still not RFC 7233 compliant.
```

Comment:
Currently, Docker Registry HTTP API V2 spec is succeeded by distribution-spec. Even DockerHub states that they conform the distribution-spec instead of the Docker Registry HTTP API V2 spec.

See https://docs.docker.com/registry/